content/en/examples/security/kind-with-cluster-level-baseline-pod-security.sh

```diff
@@ -68,3 +69,14 @@ spec:
         - containerPort: 80
 EOF
 kubectl apply -f -
+
+# Wait
+sleep 3
+
+# Clean up
+printf "\n\nCleaning up:\n" 1>&2
+set -e
+kubectl delete pod --all -n example --now
+kubectl delete ns example
+kind delete cluster --name psa-with-cluster-pss
```

Comment:
nit: Do you need to clean up the pods and namespaces if you're deleting the cluster? Also, I think deleting the pods is redundant with deleting the namespace.

Reply:
If things go wrong, the user then sees _what_ has gone wrong.